src/api/app/models/package.rb

```diff
@@ -1080,13 +1080,18 @@ def developed_packages
     packages
   end
 
-  def self.valid_name?(name)
+  def self.valid_multibuild_name?(name)
+    valid_name?(name, true)
+  end
+
+  def self.valid_name?(name, allow_multibuild = false)
```

Comment:
NITPICK: 
Why don't you use a named param instead of that second param?

I mean, using `valid_name?('package', allow_multibuild: true)` isntead of `valid_name?('package', true)`, that way from outside the method you know what is that second param for and at the same time you won't need the method `valid_multibuild_name?`.

You can implement it like:
```ruby
def self.valid_name?(name, options = {})
...
return name =~ /\A([a-zA-Z0-9]|(_product:|_patchinfo:)\w)[-+:\w\.]*\z/ ? true : false if options[:allow_multibuild]
...
end
```

Reply:
yes, I can change that.